src/network/announcethread.py

```diff
@@ -5,9 +5,9 @@
 
 # magic imports!
 import state
+import network
```

Comment:
again here we could use `from network import connectionpool`

Reply:
same reason as above, and other thing it is also throwing import error